handlers/getCategories.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const _ = require('lodash');
+const fs = require('fs');
+const path = require('path');
+const filePath = path.join(__dirname, './data.json');
+
+const getCategories = async (request, reply) => {
+    const categories = [];
+    fs.readFile(filePath, (err, data) => {
+        if(err) throw err;
+        const dataItems = JSON.parse(data);
+        dataItems.map(item => {
+            categories.push(item.category);
```

Comment:
As you only want unique items, this could be a Set instead of an array so you don't need to filter the unique values after populating the categories